Package.swift

```diff
@@ -170,32 +170,20 @@ let package = Package(
             name: "XcodeprojTests",
             dependencies: ["Xcodeproj", "TestSupport"]),
     ],
+    products: [
+        // Runtime Library: contains the package description API itself.
+        .Library(
+            name: "PackageDescription",
+            type: .dynamic,
+            targets: ["PackageDescription"]),
+
+        // SwiftPM Library: provides package management functionality to clients.
+        .Library(
+            name: "SwiftPM",
+            type: .dynamic,
+            targets: ["libc", "POSIX", "Basic", "Utility", "SourceControl", "PackageDescription", "PackageModel", "PackageLoading", "Get", "PackageGraph", "Build", "Xcodeproj", "Workspace"]),
+    ],
```

Comment:
Yup! Btw do you think we should have type as `.Dynamic` instead of `.dynamic` considering we use upper case enums everywhere in the current `PackageDescription` API ?
Right now I followed what was in the proposal